### PR TITLE
fix(dropdown/basic): style dropdown basic to be more like monolith

### PR DESCRIPTION
**Pull request by devtoni**
Modification of dropdown basic to be more like the monolith dropdown.

**mobile size:** 

Current react 
current-react

Current w/ modifications

current

Current monolith

current-modification

**desktop size:** 

Current react

captura de pantalla 2017-10-25 a las 16 03 50

Current w modif

captura de pantalla 2017-10-25 a las 16 06 29

Current monolith

captura de pantalla 2017-10-25 a las 16 03 41

### CODE REVIEW

File: components/dropdown/basic/src/index.scss [hunk 1]
```diff
@@ -134,18 +137,26 @@
       @include reset-list;
 
       &Item {
-        &:not(:first-child) {
-          margin-top: $m-v;
+        @include media-breakpoint-down(s) {
+          margin-bottom: $m-v-large;
```
Comment (davidbarna) [inline components/dropdown/basic/src/index.scss]: This is an old variable. $m-m ?

Comment (devtoni) [reply components/dropdown/basic/src/index.scss]: yes it is, I will change this variable and other old variables from this component :+1:

**Review by JoanClaret — APPROVED** [review]
great! 🎉

**Review by siscu — APPROVED** [review]
👍

**Pull request merged**